transactions/epoch/admin/set_bonus_tokens.cdc

```diff
@@ -0,0 +1,15 @@
+
+/// Saves an amount to storage that represent the amount
+/// of bonus tokens that have yet to be burned
+/// This amount is subtracted from the total supply of FLOW
+/// whenever a new total rewards amount is calculated every epoch
+/// because they are not meant to be a part of the total supply
+/// 
+/// Eventually, all bonus tokens will be burned and the amount will be zero
+
+transaction(bonusTokenAmount: UFix64) {
+    prepare(signer: AuthAccount) {
+        signer.load<UFix64>(from: /storage/FlowBonusTokenAmount)
+        signer.save(bonusTokenAmount, to: /storage/FlowBonusTokenAmount)
+    }
+}
```

Comment:
If the bonus tokens are stored in a relatively small number of known accounts, could we hard-code those addresses, and read their balance, instead of manually keeping this field in sync? 

I'm not sure how the bonus tokens are stored, so not sure how feasible this is. It would be nice to have the `bonusTokens` value stay in sync automatically, though, if possible.

Reply:
I agree, but I think that they are stored in a bunch of accounts. That is a good suggestion though. I'm asking to see if that would be possible

Reply:
We discussed more and we're going to keep it this way because monitoring the different accounts was going to be more operationally and technically difficult. We think requiring the multisig is good for this because we can just do it at the same time as the burn